docs/src/template.js

```diff
@@ -30,21 +30,27 @@ export default ({ head = {}, html = '', assets = {} }) => `<!doctype html>
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <meta name="robots" content="noindex">
 
-  <meta property="og:locale" content="en_GB">
   <meta name="description" content="Backpack is Skyscanner's open-source design system.">
+
   <meta property="og:description" content="Backpack is Skyscanner's open-source design system.">
-  <meta property="og:url" content="https://backpack.github.io">
-  <meta property="og:site_name" content="Backpack">
   <meta property="og:image" content="https://backpack.github.io/social-preview.png">
   <meta property="og:image:width" content="256">
   <meta property="og:image:height" content="256">
+  <meta property="og:locale" content="en_GB">
+  <meta property="og:site_name" content="Backpack">
+  <meta property="og:title" content="Backpack">
+  <meta property="og:type" content="website">
+  <meta property="og:url" content="https://backpack.github.io">
+
   <meta property="twitter:card" content="summary">
+  <meta property="twitter:creator" content="@skyscanner">
   <meta property="twitter:image" content="https://backpack.github.io/social-preview.png">
-  <meta property="twitter:title" content="Backpack">
+  <meta name="twitter:image:alt" content="Backpack — Skyscanner's design system">
```

Comment:
Should this be `name`?

Reply:
Ah, yeah this is a really good point. I copypasta'd from here: https://css-tricks.com/essential-meta-tags-social-media/

<img width="937" alt="Screen Shot 2020-12-11 at 10 10 10" src="https://user-images.githubusercontent.com/73652/101890766-0ed7f080-3b99-11eb-9660-284f4d90eb54.png">

Having said that, we know the ones we have work fine with `property` so I'll change this one too, for consistency.